.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+
+      - name: Build compliance-masonry
+        run: go build cmd/compliance-masonry/compliance-masonry.go
+
+      - name: Build masonry
+        run: go build cmd/masonry/masonry.go
+
+      - name: Run tests
+        run: |
+          go test $(go list ./...) -covermode=count -coverprofile=coverage.txt
+          go tool cover -func coverage.txt
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
syntax here looks good to me; I assume we'd keep this conditional to only perform codecov for `ubuntu-latest` even if the other OS's (windows,macos-latest) were included here, too? (It should be the same in any case regardless of OS I'd have to assume, no need to upload x3 times, right?). regardless, lgtm

Reply:
exactly